controller/work-item-link-blackbox_test.go

```diff
@@ -289,7 +314,7 @@ func CreateWorkItem(workItemType satoriuuid.UUID, title string) *app.CreateWorki
 }
 
 // CreateWorkItemLinkType defines a work item link type
-func CreateWorkItemLinkType(name string, sourceTypeID, targetTypeID, categoryID, spaceID satoriuuid.UUID) *app.CreateWorkItemLinkTypePayload {
+func CreateWorkItemLinkType(name string, sourceTypeID, targetTypeID, categoryID, spaceID uuid.UUID) *app.CreateWorkItemLinkTypePayload {
```

Comment:
Well, I agree with that cause I always ask about it. But this change was already started in many other files where I had to solve conflicts. So I had to resolve them, consequently decided to apply those changes to the files I touched. 
Maybe we should try to avoid those partial changes in the future.